corehq/apps/users/role_utils.py

```diff
@@ -23,16 +24,38 @@ class UserRolePresets:
         BILLING_ADMIN: lambda: HqPermissions(edit_billing=True),
         MOBILE_WORKER: lambda: HqPermissions(access_mobile_endpoints=True,
                                              report_an_issue=True,
-                                             access_all_locations=True)
+                                             access_all_locations=True),
     }
 
+    PRIVILEGED_ROLES = {
+        PROGRAM_MANAGER: lambda: HqPermissions(
+            edit_groups=True,
+            view_groups=True,
+            edit_users_in_groups=True,
+            edit_data=True,
+            edit_apps=True,
+            view_apps=True,
+            access_web_apps=True,
+            edit_reports=True,
+            download_reports=True,
+            view_reports=True
```

Comment:
(Probably a question for Norman, since this is design related), why not create an other permission instead of a role for ATE management such as edit_ate? @kaapstorm

Reply:
Just noting here what we discussed in the ticket discussion: we'll be creating a specific permission and default role for people to use.